src/studio/src/designer/frontend/app-development/package.json

```diff
@@ -56,13 +56,13 @@
     "@types/react-router-dom": "~5.1.7",
     "@types/react-test-renderer": "~17.0.1",
     "@types/redux-mock-store": "~1.0.2",
-    "altinn-designsystem": "2.7.12",
-    "babel-jest": "~27.0.2",
+    "altinn-designsystem": "3.1.0",
```

Comment:
We can't update altinn-designsystem in studio, as there are some font issues that need to be resolved first for studio: https://github.com/Altinn/altinn-studio/issues/5824
So for all studio applications, the version needs to be `2.7.12`.